src/indexes/customIndexes.js

```diff
@@ -353,7 +353,7 @@ module.exports = {
                     '_uuid': 1
                 },
                 options: {
-                    name: 'health_notifications'
+                    name: 'communication.health_notifications'
```

Comment:
Real problems are these keys
- "*"
- "*-history"

So, either add "collections name with those collections" or remove the collection name from everywhere. 
Also, whatever decided, update index format in the readme.